aws/resource_aws_ec2_capacity_reservation_test.go

```diff
@@ -0,0 +1,222 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func init() {
+	resource.AddTestSweepers("aws_ec2_capacity_reservation", &resource.Sweeper{
+		Name: "aws_ec2_capacity_reservation",
+		F:    testSweepEc2CapacityReservations,
+	})
+}
+
+func testSweepEc2CapacityReservations(region string) error {
+	client, err := sharedClientForRegion(region)
+	if err != nil {
+		return fmt.Errorf("error getting client: %s", err)
+	}
+	conn := client.(*AWSClient).ec2conn
+
+	resp, err := conn.DescribeCapacityReservations(&ec2.DescribeCapacityReservationsInput{})
+
+	if err != nil {
+		return fmt.Errorf("Error retrieving capacity reservations: %s", err)
+	}
+
+	if len(resp.CapacityReservations) == 0 {
+		log.Print("[DEBUG] No capacity reservations to sweep")
+		return nil
+	}
+
+	for _, r := range resp.CapacityReservations {
+		if *r.State != "cancelled" {
+			id := aws.StringValue(r.CapacityReservationId)
+
+			log.Printf("[INFO] Cancelling capacity reservation EC2 Instance: %s", id)
+
+			opts := &ec2.CancelCapacityReservationInput{
+				CapacityReservationId: aws.String(id),
+			}
+
+			_, err := conn.CancelCapacityReservation(opts)
+
+			if err != nil {
+				log.Printf("[ERROR] Error cancelling capacity reservation (%s): %s", id, err)
+			}
+		}
+	}
+
+	return nil
+}
+
+func TestAccAWSEc2CapacityReservation_importBasic(t *testing.T) {
```

Comment:
Instead of separate import acceptance test functions, we prefer to include the `ImportState` `TestStep` in all acceptance tests. 👍